libs/utils/src/filenames.ts

```diff
@@ -108,88 +59,6 @@ export function generateElectionBasedSubfolderName(
   )}`;
 }
 
-/**
- * Generate the directory name for the cast vote record report.
- *
- * @deprecated
```

Comment:
Thanks for removing these! Forgot about them when I was cleaning up old CVR code